### ISSUE TITLE
Simple string headers for generated format

### ISSUE THREAD

**Issue by SylvanoTombo** [FriendsOfSylius/SyliusImportExportPlugin#244]
Hi,
Actually headers in generated file is using id dynamically, how can i use a simple string in the headers?

**Comment by oallain** [comment]
Hello,

I don't think it"s configurable, it's based on https://github.com/FriendsOfSylius/SyliusImportExportPlugin/blob/7aa80b422e56c8d83918a207ba612be7f279abea/src/Exporter/ResourceExporter.php#L31

For exemple : https://github.com/FriendsOfSylius/SyliusImportExportPlugin/blob/7aa80b422e56c8d83918a207ba612be7f279abea/src/Resources/config/services_export_csv.yml#L18

**Comment by SylvanoTombo** [comment]
@oallain , Yeah but the resourceKeys must be in fields in the entity but not a simple string

**Comment by oallain** [comment]
Yes, sorry.
Do you want to make a Pull Request for add this feature?

**Comment by SylvanoTombo** [comment]
Hi, 
I think there are a function for that like "exportData" but the ExportDataController use the export function by default.
